include/cutlass/gemm/kernel/intel_pvc_gemm.hpp

```diff
@@ -0,0 +1,286 @@
+    int thread_idx = int(ThreadIdxX());
+    auto subgroup_shape = TileShape{};                                                  // (SUB_M,SUB_N,SUB_K)
+    const int m_coord = BlockIdxX() * get<0>(subgroup_shape);
+    const int n_coord = (BlockIdxY() * num_sg + thread_idx / SubgroupSize) * get<1>(subgroup_shape);
+    const int l_coord = BlockIdxZ();
+
+    Tensor tAi = params.mainloop.gmem_tiled_copy_a.get_pvc_tensor(make_coord(m_coord, 0, l_coord),
+                                                                  make_shape(_1{}, K, L),
+                                                                  make_stride(Int<FragsM * DpasM>{}, _1{}));
+
+    Tensor tBi = params.mainloop.gmem_tiled_copy_b.get_pvc_tensor(make_coord(0, n_coord, l_coord),
+                                                                  make_shape(K, Int<FragsN>{}, L),
+                                                                  make_stride(_1{}, Int<DpasN>{}));
+
+    // Compute tile residues for predication
+    auto m_max_coord = M - get<0>(subgroup_shape) * m_coord;                             // M - SUB_M * m_coord
+    auto n_max_coord = N - get<1>(subgroup_shape) * n_coord;                             // N - SUB_N * n_coord
+    auto k_residue   = K - get<2>(subgroup_shape) * (K / get<2>(subgroup_shape));        // K - SUB_K * k_coord_max
+    auto residue_mnk = make_tuple(m_max_coord, n_max_coord, k_residue);
+
+    // Allocate the tiled_mma and the accumulators for the (M,N) subgroup_shape
+    TiledMma tiled_mma;
+
+    Tensor accumulators = make_tensor<float>(Shape<Int<VecC>, Int<FragsM>, Int<FragsN>>{});
+    clear(accumulators);
+
+    auto k_tile_iter  = cute::make_coord_iterator(make_shape(K / get<2>(subgroup_shape)));
+    int  k_tile_count = K / get<2>(subgroup_shape);
+
+    // Perform the collective scoped MMA
+    CollectiveMainloop collective_mma;
+    collective_mma(
+      accumulators,
+      tAi(_,_,_,l_coord),
+      tBi(_,_,_,l_coord),
+      accumulators,
+      k_tile_iter, k_tile_count,
+      residue_mnk,
+      thread_idx,
+      smem_buf,
+      params.mainloop
+    );
+    auto gmem_tiled_copy_c = make_xe_2d_copy<XE_2D_U32x8x16x1x1_ST_N>(make_tensor(params.epilogue.ptr_D, make_shape(M, N, L), params.epilogue.dD));
+
+    Tensor tCi = gmem_tiled_copy_c.get_pvc_tensor(make_coord(m_coord, n_coord, l_coord),
+                                                  make_shape(Int<FragsM>{}, Int<FragsN>{}, L),
+                                                  make_stride(Int<DpasM>{}, Int<DpasN>{}));
+
+    copy(gmem_tiled_copy_c, accumulators, tCi(_,_,_,l_coord));
+
+    // Epilogue and write to gD
+    // CollectiveEpilogue epilogue{params.epilogue};
+    // epilogue(
+    //   problem_shape_MNKL,
+    //   sg_shape{},
+    //   blk_coord_mnkl,
+    //   accumulators,
+    //   tiled_mma,
+    //   residue_mnk,
+    //   thread_idx,
+    //   smem_buf
+    // );
```

Comment:
Why is this commented if it is not needed please remove it. If it is needed please enable it

Reply:
This is commented since we don't have support for the Epilogue yet, once we add that then this code is going to be uncommented and lines `261-267` will get removed.

Reply:
Sam as above, if it is not use it should be removed and can can be added when there is epilogue